### PR TITLE
Add development guide

### PR DESCRIPTION
**Pull request by zhouqiang-cl**
### What problem does this PR solve? 
Add development guide
### What is changed and how does it work?
Add development guide
### Check List 

Tests 

 - No code

Code changes

 - None

Side effects

 - None

Related changes

 - None

### CODE REVIEW

File: doc/DEVELOP.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Development documentation
+
+This section of the documentation is intended to get you to develop Chaos Mesh project. We'll cover things you need to know, from setup environment to add some kind of chaos.
```
Comment (mahjonp) [inline doc/DEVELOP.md]:
```suggestion
This tutorial will walk you through being a Chaos Mesh project developer from scratch, and cover something you're interested in.
```

File: doc/DEVELOP.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Development documentation
+
+This tutorial will walk you through being a Chaos Mesh project developer from scratch, and cover something you're interested in.
```
Comment (mapleFU) [inline doc/DEVELOP.md]: Maybe `you may` is better than `you're` ?

**Comment by cwen0** [comment]
@dcalvin PTAL

File: doc/DEVELOP.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
```
Comment (dcalvin) [inline doc/DEVELOP.md]:
```suggestion
This guide prepares you for the development of Chaos Mesh from scratch. Before you get started, it is recommended you get familiar with the project through the following materials:

- [README](./README.md)
```

File: doc/DEVELOP.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+
+The tutorial is listed here
```
Comment (dcalvin) [inline doc/DEVELOP.md]:
```suggestion
The development flow starts from [Set up your development environment]. After this, you can choose any of the following procedures to contribute:
```

File: doc/DEVELOP.md [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+- [ ] Set up your development environment
```
Comment (dcalvin) [inline doc/DEVELOP.md]:
```suggestion
```

**Pull request merged**